store_queue.py

```diff
@@ -0,0 +1,55 @@
+import pika
+import settings
+
+
+class StoreQueue(object):
```

Comment:
I think we should rename this class something like QueuePublisher. We should be able to re-use this in sdx-downstream.